bots/images/scripts/fedora.setup

```diff
@@ -27,6 +27,11 @@ if [ $(uname -m) = x86_64 ]; then
     HAVE_KUBERNETES=1
 fi
 
+HAVE_LIBVIRT_DBUS=
+if ! [[ $IMAGE =~ ^fedora\-.*[stock|i386|atomic]$ ]]; then
```

Comment:
Is libvirt-dbus really a 64 bit only thing? If not, then this condition is not necessary at all, you can just statically add libvirt-dbus to the dependency list. fedora-{atomic,stock} have their own .setup scripts. If libvirt-dbus is really not available for i386, then just filter for that.

Reply:
Actually it is available, https://koji.fedoraproject.org/koji/buildinfo?buildID=1082654.
Removed condition completely.